README.md

```diff
@@ -1 +1,52 @@
-# TO-do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+> "To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+This website is built using ES6 and Webpack!
+
+## Below is a PNG showing the app
+![PNG](./Screenshot.png)
+
+## Live Demo
+
+You can see [the app live here](https://sahar-abdelsamad.github.io/To-do-list/)
```

Comment:
- [ ] **Optional:** The live demo link opens to the README, it would be nice to see a demo of your app,  what do you think?

Reply:
Yes, sure in fact this is what I am trying to do but since I am using webpack the live link came like that.
Do you know if I can fix it? Can it be linked to the HTML file inside the dist?

Reply:
Done :+1: